### PR TITLE
Fee splitter: initial skeleton

### LINKED ISSUE #44: Create new contract skeleton
Create a skeleton for a new contract called `fee_splitter`

### PR DESCRIPTION
**Pull request by gangov**

### CODE REVIEW

**Comment by ueco-jb** [comment]
Good start, fix the lints and you can merge it.

File: contracts/fee_splitter/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+[package]
+name = "fee_splitter"
```
Comment (ueco-jb) [inline contracts/fee_splitter/Cargo.toml]:
```suggestion
name = "fee-splitter"
```

File: contracts/fee_splitter/.cargo/config [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[alias]
+wasm = "build --release --lib --target wasm32-unknown-unknown"
+wasm-debug = "build --lib --target wasm32-unknown-unknown"
+unit-test = "test --lib"
+integration-test = "test --test integration"
+schema = "run --bin pool_schema"
```
Comment (ueco-jb) [inline contracts/fee_splitter/.cargo/config]: This will be something else.

Comment (gangov) [reply contracts/fee_splitter/.cargo/config]: should I remove it?

Comment (ueco-jb) [reply contracts/fee_splitter/.cargo/config]:
```suggestion
schema = "run --bin fee_splitter_schema"
```

File: contracts/fee_splitter/src/contract.rs [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+            // config.addresses.iter().for_each(|(address, decimal)| {
+            for (address, decimal) in config.clone().addresses.into_iter() {
+                let send_amount = amount * decimal;
+                let msg = CosmosMsg::Bank(BankMsg::Send {
+                    to_address: address.clone().to_string(),
+                    amount: vec![Coin {
+                        denom: denom.to_string(),
+                        amount: send_amount,
+                    }],
+                });
```
Comment (ueco-jb) [inline contracts/fee_splitter/src/contract.rs]:
```suggestion
                let msg = CosmosMsg::Wasm(WasmMsg::Execute {
                    contract_addr: denom.clone(),
                    msg: to_binary(&Cw20ExecuteMsg::Transfer { recipient: address.clone(), amount: send_amount })?,
                    funds: vec![],
                });
```

And for that to work you need to import `use cw20::Cw20ExecuteMsg;`

Comment (ueco-jb) [reply contracts/fee_splitter/src/contract.rs]: Fixed.

**Pull request merged**